homeworks/karolinavalko/4_exceptions_logging/json_to_do.py

```diff
@@ -0,0 +1,84 @@
+import logging
+import logging.config
+import os
+import json
+
+print(os.getcwd())
+
+logging.config.fileConfig(
+    r"c:\Users\tabal\OneDrive\ドキュメント\python_hu\homeworks\karolinavalko\4_exceptions_logging\logger_set_up.ini"
+)
+logger = logging.getLogger("my_logger")
+
+file_path = r"c:\Users\tabal\OneDrive\ドキュメント\python_hu\homeworks\karolinavalko\4_exceptions_logging\to_do.json"
+to_do_list = []
+display_menu = ["Add task", "View task", "Remove task", "Exit"]
+
+
+def read_file(file_path):
+    try:
+        with open(file_path, "r") as file:
+            logger.info("File had been opened in read mode.")
+            return json.load(file)
+    except FileNotFoundError as e:
+        logger.warning(f"The file {file_path} does not exist, creating one.")
+        write_list(file_path, [])
+        return []
+    except:
+        logger.error(f"The file {file_path} could not be read.")
+
+
+def view_items():
+    to_do_list = read_file(file_path)
+    for number, item in enumerate(to_do_list, 1):
+        print(f"{number}. {item.strip()}")
+    logger.info("To do list had been viewed.")
+    return to_do_list
+
+
+def write_list(file_path, list):
+    with open(file_path, "w") as file:
+        json.dump(list, file, indent=2)
+
+
+def add_item():
+    to_do_list = read_file(file_path)
+    new_item = input("Please type what do you want to add:")
+    to_do_list.append(new_item)
+    write_list(file_path, to_do_list)
+    logger.info(f"{new_item} has been successfully added to the To Do list.")
+    print(f"You added {new_item} to the list.")
+
+
+def remove_item():
+    try:
+        to_do_list = view_items()
+        to_remove = int(input("Please select task to remove:"))
+        del to_do_list[to_remove - 1]
+        write_list(file_path, to_do_list)
+        logger.info(f"{to_remove} have been successfully removed from the list.")
+    except IndexError as e:
+        logger.error(f"Index error: {e}")
+        print("This item does not exist. Choose another task to be removed")
+
+
+while True:
```

Comment:
ez az egész loop bekerülhetne egy main() function-be, ezzel jelezve hogy ez a kódod kiinduló pontja.